File: drs/src/test/com/db4o/drs/test/data/Value.java
package com.db4o.drs.test.data;

/**
 * @sharpen.struct
 */
public class Value
{
	public int value;
	
	public Value(int value) {
		this.value = value;
	}
	
	public boolean equals(Object obj) {
		if (!(obj instanceof Value)) {
			return false;
		}
		Value other = (Value)obj;
		return other.value == value;
	}
}

File: db4o.net/drs/Db4objects.Drs.Tests/Db4objects.Drs.Tests/Data/Value.cs
/* Copyright (C) 2004 - 2011  Versant Inc.  http://www.db4o.com */

namespace Db4objects.Drs.Tests.Data
{
	public struct Value
	{
		public int value;

		public Value(int value)
		{
			this.value = value;
		}

		public override bool Equals(object obj)
		{
			if (!(obj is Db4objects.Drs.Tests.Data.Value))
			{
				return false;
			}
			Db4objects.Drs.Tests.Data.Value other = (Db4objects.Drs.Tests.Data.Value)obj;
			return other.value == value;
		}
	}
}
